util/deploy-production.sh

```diff
@@ -73,5 +74,64 @@ then
     exit 2
 fi
 
-# TODO(past): automate the remaining steps
-echo "Until the process if fully automated, continue with the manual deployment process starting at step 4."
+# Confirm there are 3 versions for each service.
+SERVICES="default processor searchcache"
+for SERVICE in $SERVICES
+do
+  VERSIONS=$(gcloud app --project=wptdashboard versions list --filter="service=$SERVICE" | wc -l)
+  # The command above should print a header line and one line per version.
+  if [[ "${VERSIONS}" != "4"  ]];
```

Comment:
I wonder if we should compare greater than 3 instead of strictly equal to 3. The case I'm wondering about: What happens if this script fails part way through and we have to re-run.

An example that happened to me once: App Engine had an intermittent failure where the initial deployment failed. Nothing wrong with the code but I just had to start the deployment again and it worked. If that case comes up again, the oldest version would be deleted leaving you less than 3 on retry.

Reply:
An option to help with that:
- Combine the `# Confirm there are 3 versions for each service.` and `# Delete the oldest versions.` into one section. Some pseudocode
  - ```psuedocode
    if VERSIONS == 3
       delete the oldest
    if VERSIONS < 3
        don't delete anything
    else
        print that you found too many versions and exit
    ```

Reply:
I like this idea.